### DISCUSSION TITLE
Welcome Message

### DISCUSSION THREAD

**Discussion by sami1800** [sami1800/Software-Requirement-Semester-Project#2] [Announcements]
I welcome you all to the project.
As everyone is well aware of your role so we will start our project as soon as possible.

**Comment by leekonjk** [comment]
Hurry

**Reply by sami1800** [reply]
Be serious other wise I will kick you out

**Reply by leekonjk** [reply]
Wow

**Reply by sami1800** [reply]
Insan bun na yeh kuch kr maron ga

**Reply by leekonjk** [reply]
Aga kya kar na ha

**Reply by sami1800** [reply]
jb krna ho ga btion ga wakti so ja
